bin/wasm-node/javascript/src/index-browser.ts

```diff
@@ -256,29 +256,29 @@ export function start(options?: ClientOptions): Client {
               "v=0" + "\n" +
               // Identifies the creator of the SDP document. We are allowed to use dummy values
               // (`-` and `0.0.0.0`) to remain anonymous, which we do. Note that "IN" means
-              // "Internet". (RFC8866)
+              // "Internet" (and not "input"). (RFC8866)
               "o=- 0 0 IN IP" + ipVersion  + " " + targetIp + "\n" +
               // Name for the session. We are allowed to pass a dummy `-`. (RFC8866)
               "s=-" + "\n" +
               // Start and end of the validity of the session. `0 0` means that the session never
               // expires. (RFC8866)
               "t=0 0" + "\n" +
-              // A lite implementation is only appropriate for devices that will
-              // *always* be connected to the public Internet and have a public
+              // A non-lite implementation is only appropriate for devices that will
```

Comment:
??? This was a copy from the specification. Did something change?

Reply:
? The specification doesn't contain this text. It's me who wrote it. It has a mistake in it, it says that lite implementations are only for publicly-accessible devices. But no, it's non-lite implementations that are for public devices. We're not a public device, so we use the lite implementation.

Reply:
"Certain ICE agents will always be connected to the public Internet
   and have a public IP address at which it can receive packets from any
   correspondent.  To make it easier for these devices to support ICE,
   ICE defines a special type of implementation called "lite" (in
   contrast to the normal full implementation).  Lite agents only use
   host candidates and do not generate connectivity checks or run state
   machines, though they need to be able to respond to connectivity
   checks." https://datatracker.ietf.org/doc/rfc8445/

Reply:
By specifying `ice-lite` in server's SDP (that we're pretending we've received through some channel, but instead construct manually), we're saying that server is publicly accessible.

Reply:
To be honest this text doesn't make sense to me no matter whether it says "lite" or "not lite", so I've just reverted it :shrug: